.github/workflows/qit-environment-test-linux.yml

```diff
@@ -1,15 +1,121 @@
-name: QIT Environment Test - Linux
+name: QIT Environment Test - Linux and Mac
 
 on:
+  # Every day at 11pm UTC (6pm ET)
+  schedule:
+    - cron: '0 23 * * *'
   # Manually
   workflow_dispatch:
 
 jobs:
   environment_tests:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macos-13 ]
+        php: [ '7.4' ]
+        wordpress: [ 'rc' ]
+        woocommerce: [ 'rc' ]
+        include:
+          - os: ubuntu-20.04
+            php: '8.3'
+            wordpress: 'rc'
+            woocommerce: 'rc'
+      fail-fast: false
     env:
       NO_COLOR: 1
       QIT_DISABLE_ONBOARDING: yes
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
\ No newline at end of file
+      - name: Checkout code (Cross-platform)
+        uses: actions/checkout@v4
+
+      - name: Setup PHP (Cross-platform)
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.3'
+
+      - name: Enable dev mode (Cross-platform)
+        run: ./qit dev
+
+      - name: Connect to Staging QIT (Cross-platform)
+        run: ./qit backend:add --environment="staging" --qit_secret="${{ secrets.QIT_STAGING_SECRET }}" --manager_url="https://stagingcompatibilitydashboard.wpcomstaging.com"
+
+      - name: Add "qit.test" to hosts file (Linux)
+        if: matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-13'
+        run: sudo echo "127.0.0.1 qit.test" | sudo tee -a /etc/hosts
+
+      - name: Setup Docker on macOS (Mac)
+        if: matrix.os == 'macos-13'
+        uses: douglascamata/setup-docker-macos-action@v1-alpha
+
+      - name: Check Colima Status (Mac)
+        if: matrix.os == 'macos-13'
+        run: colima status
+
+      - name: Debug Docker and Docker Compose (Unix)
+        if: matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-13'
+        run: |
+          echo "Checking Docker version..."
+          docker --version || echo "Docker is not installed"
+
+          echo "Checking Docker Compose (plugin) version..."
+          docker compose version || echo "Docker Compose (plugin) is not installed"
+
+          echo "Checking Docker Compose (standalone) version..."
+          docker-compose --version || echo "Docker Compose (standalone) is not installed"
+
+      - name: Benchmark Docker (Unix)
```

Comment:
Looks like this step download WordPress. Is Benchmark the best name to use here?

Reply:
Yes, it's benchmarking how long does it take to download and extract WordPress. This is very fast on Linux, but takes ages on Mac in CI.